packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -134,6 +152,27 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
           </TrimWrapper>
         )}
       </Row>
+      {showVolumeControl && (
+        <VolumeWrapper>
+          <Text
+            as="label"
+            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+            htmlFor={slideId}
+          >
+            {__('Volume', 'web-stories')}
+          </Text>
+          <StyledSlider
+            value={Math.round(volume * 100)}
+            handleChange={onChangeVolume}
+            minorStep={5}
+            majorStep={10}
+            min={0}
```

Comment:
```suggestion
            min={10}
```

Otherwise it's impossible to get more to the left:

<img width="352" alt="Screenshot 2022-09-13 at 10 26 06" src="https://user-images.githubusercontent.com/841956/189969568-6fd80651-0f4e-4baf-805d-fd230b228efd.png">

Reply:
Yeah, I tried this and sadly it did not work, as it generated some weird UX. 

https://user-images.githubusercontent.com/237508/190159911-1d33addd-b725-4f5c-b05e-67b944c01498.mov


The slider is at the very end, but is still 10, that is confusing. If the slider is at the start, it should be 0. The way it was written, it would not let you set it to 0, but you could see that a little bit of the slider was across. 

Maybe we should just allow 0 volume. If set, either show a tooltip recommending you remove the audio or just trigger the removal of audio automatically.

Reply:
Either way is confusing right now, yeah, but I we'll probably replace the numbers with something else anyway. So don't worry about it for now. It's just the first PR of many. We'll finalize UX later.

Reply:
I personally think that the way that it is now, is the best way. We can change it in a follow up PR.

Reply:
We'll definitely change this in a followup.